esy-lib/Fs.ml

```diff
@@ -278,6 +278,7 @@ let rec rmPathLwt path =
 
     Lwt_unix.rmdir pathS
   | _ ->
+    let%lwt () = Lwt_unix.chmod pathS 0o640 in
```

Comment:
On Windows, some of the files in the `.git` folder (which we delete with one of the sources strategies) were marked as 'read-only'. The permission model is a bit different on Windows, but this call does actually work for files to remove the read-only permission - and allow the deletion / unlink to complete successfully.